packages/server/lib/util/find_system_node.js

```diff
@@ -41,44 +38,6 @@ function findNodeInFullPath () {
   })
 }
 
-function findNodeVersionFromPath (path) {
-  if (!path) {
-    return Promise.resolve(null)
-  }
-
-  return execa.stdout(path, ['-v'])
-  .then((stdout) => {
-    debug('node -v returned %o', { stdout })
-    const matches = NODE_VERSION_RE.exec(stdout)
-
-    if (matches && matches.length === 2) {
-      const version = matches[1]
-
-      debug('found Node version', { version })
-
-      return version
-    }
-  })
-  .catch((err) => {
-    debug('could not resolve Node version %o', { err })
-
-    throw err
-  })
-}
-
-function findNodePathAndVersion () {
-  return findNodeInFullPath()
-  .then((path) => {
-    return findNodeVersionFromPath(path)
-    .then((version) => {
-      return {
-        path, version,
-      }
-    })
-  })
-}
-
 module.exports = {
   findNodeInFullPath,
```

Comment:
Ahh crap. Was gonna ask why this is here, but I forgot that `findNodeInFullPath` is used in the `darwin` `child_process` spawn workaround. We only use it in one place: https://github.com/cypress-io/cypress/blob/f8fd333403c8199277b6be81e8ffe913d6351edd/packages/launcher/lib/darwin/util.ts#L108-L119

The reason we need to use the system node here is a bug in Electron that causes cp.spawn to be very slow on `darwin`, which causes our browser detection to be very slow. I guess it's not an option to remove `node-which` entirely yet then since we'd have to make performance bad for global-mode users on `darwin` during browser detection.